lib/src/common/cache/most_recently_used_strategy.dart

```diff
@@ -0,0 +1,49 @@
+import 'dart:async';
+import 'dart:collection';
+
+import 'package:w_common/cache.dart';
+
+/// A [CachingStrategy] that will hold the last n most recently used [TValue]s.
+///
+/// When strategy is constructed with 0 most recently used values held in the
```

Comment:
This comment is confusing - `When this strategy is used to construct a cache with 0 most recently used values, it always removes the cache on release`?

Reply:
What about `When n = 0 the strategy will remove on every release`?

Reply:
way better, just think it's worth clarifying what is removed. `When n = 0 the strategy will remove the value on every release` ?